src/info_dump/main.cpp

```diff
@@ -166,5 +166,32 @@ extern "C" __declspec(dllexport) void run(DWORD pid)
         textures_file.write(dump.data(), dump.size());
     }
 
+    if (std::ofstream search_flags_file = std::ofstream("game_data/search_flags.json"))
+    {
+        float_json search_flags(float_json::object());
+        for (size_t i = 31; i < 32; i--)
+        {
+            auto items = list_entities();
+            std::sort(items.begin(), items.end(), [](EntityItem &a, EntityItem &b) -> bool { return a.id < b.id; });
```

Comment:
[pedantic] listing entities and sorting them could be placed before the for loop

Reply:
Uhh yes, could even be done fore all the json stuff. Will fix.